server/src/utils/placement_entry_util.js

```diff
@@ -0,0 +1,69 @@
+const JobLeadTimelineEntry = require("../models/job_lead_timeline_entry.model");
+const EmployerTimelineEntry = require("../models/employer_timeline_entry.model");
+const ClientTimelineEntry = require("../models/client_timeline_entry.model");
+const JobLead = require("../models/job_lead.model");
+const Client = require("../models/user.model");
+const User = require("../models/user.model");
+
+const submitPlacementUpdateEntryInTimelines = async (data, pageType) => {
+  // eslint-disable-next-line camelcase
+  const { type, job_lead, client, user, employer } = data;
+
+  // eslint-disable-next-line camelcase
+  const jobLeadObject = await JobLead.findOne({ where: { id: job_lead } });
+  const clientObject = await Client.findOne({ where: { id: client } });
+  const userObject = await User.findOne({ where: { id: user } });
```

Comment:
user is now a user object, so this query won't work

Reply:
im passing in user as the id not the object into this function.

Reply:
Ok that sounds good, but currently as its passed in within addEmployerTimelineEntry and addJobLeadTimelineEntry it is the user object that is being passed in not the user id (following from req.user returning the user object and not id of the user)
<img width="782" alt="Screenshot 2024-03-03 at 4 14 50 PM" src="https://github.com/uoftblueprint/tces/assets/58835213/64e555c0-b873-4d54-923f-c836c578887f">